src/open_samus_returns_rando/specific_patches/chozo_seal_patches.py

```diff
@@ -165,8 +166,9 @@ def update_item_seals(editor:PatcherEditor):
             scenario.raw.actors[16][seal]["type"] = "chozoseal"
 
         # Dependencies
-        for asset in editor.get_asset_names_in_folder("actors/props/chozoseal"):
-            editor.ensure_present_in_scenario(scenario_name, asset)
+        for level_pkg in editor.get_all_level_pkgs():
```

Comment:
Why all level pkgs? This should only be relevant for the scenarios where we add stuff.

Reply:
Should I list out all scenarios where we have the custom assets, or just a6 since that's where it breaks things?

Reply:
No, just look what was the code before. It was already only adding the dependencies to the scenario and it was also only using the scenario name of the currently processed chozo seal in the loop

Reply:
So this would work?
```suggestion
        for asset in editor.get_asset_names_in_folder("actors/props/chozoseal"):
            editor.ensure_present(get_package_name(scenario_name, asset), asset)
```